shim.go

```diff
@@ -94,12 +93,16 @@ func (s *shim) proxyStdio(wg *sync.WaitGroup, terminal bool) {
 }
 
 // handleSignals performs all signal handling.
-func (s *shim) handleSignals() chan os.Signal {
+//
+// The tty parameter is specific to SIGWINCH handling.
+func (s *shim) handleSignals(tty *os.File) chan os.Signal {
 	sigc := make(chan os.Signal, sigChanSize)
 	// handle all signals for the process.
 	signal.Notify(sigc)
 	signal.Ignore(syscall.SIGCHLD, syscall.SIGPIPE)
 
+	s.resizeTty(tty)
```

Comment:
@jodh-intel Why is this call to resizeTty required?

Reply:
Good question - I just carried over the logic from the original `resizeTty()` function. However, that call seems bogus as testing shows it works perfectly well without it (containers always receive a `SIGWINCH` on startup from docker/containerd fwics).